README.md

```diff
@@ -22,12 +22,27 @@
 ### Ok, I get that this is a problem, how to find out that I'm affected?
   
 * Failures in the server log of http requests that are making updates 
+Looking at the web server logs to look failures related to http requests that were doing updates. Any 4xx or 5xx errors related to PUT, POST, DELETE requests indicate that probably such issue occurred in the application.
+ 
 * Not all failures are visible in the server logs => timeout on the way back to the client
+Not every failure will be visible in the web server logs. Lost responses from web server to the client will not be logged. Lack of errors in the log does not mean that such problems never took place. 
+
 * Do you even have logs? are you checking them?
+Logs need to be configured and looked at regular basis. Certain web servers like IIS have logging configured but using console applications to host your web api might not. 
+
 * Catching and logging http related exceptions in the client
+Most of the exceptions  will be visible on the client side, and should be caught and logged. ???
```

Comment:
Isn't this the continuation of the first sentence, ie not all errors will be in the server logs so to have "full coverage" you need to catch http releated errors in the client as well?

So perhaps remove this and extend the first sentence?